src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/basic/GregtechMetaWirelessCharger.java

```diff
@@ -471,6 +470,7 @@ public void onPostTick(final IGregTechTileEntity aBaseMetaTileEntity, final long
                 this.mCurrentDimension = aBaseMetaTileEntity.getWorld().provider.dimensionId;
             }
 
+            boolean mHasBeenMapped = this.equals(ChargingHelper.getEntry(getTileEntityPosition()));
             if (!mHasBeenMapped && ChargingHelper.addEntry(getTileEntityPosition(), this)) {
                 mHasBeenMapped = true;
             }
```

Comment:
This section is run every tick, and does some map searching for every TE. While in the grand scheme of things this is a trivial difference, I would still like to see it moved below to the "every 20 ticks" conditional.

Otherwise OK, thanks for the fix!

Reply:
Thank you for the feedback. I have made the requested change.